sousmot/sousmotapp/templates/sousmotapp/game.html

```diff
@@ -0,0 +1,66 @@
+{% extends "sousmotapp/base.html" %}
+
+{% load static %}
+{% block title %}Home{% endblock %}
+
+{% block content %}
+    <section class="hero has-text-centered">
+        <div class="hero-body">
+            <h1 class="title">SousMot</h1>
+            <p class="subtitle">A Multiplayer Customizable Wordle game !</p>
+            <p class="subtitle"> {{ mode }}</p>
+            <p id="timer" class="subtitle"></p>
```

Comment:
```suggestion
            <p id="timer" class="subtitle">00:08</p>
```
Maybe add something for the start of the game before the JS is loaded ?